packages/logger/src/index.ts

```diff
@@ -8,26 +8,25 @@ const loggers: LRU<string, Debugger> = new LRU({
 });
 
 class Logger {
-  private tags: string[];
-
-  constructor(tags: string | string[]) {
-    if (tags instanceof Array) {
-      this.tags = tags;
-    } else {
-      this.tags = [tags];
-    }
+  private service: string;
+  private tags: string[] | undefined;
```

Comment:
I think you can remove `| undefined` here and set the value of `tags` to an empty array on the constructor by default.
This will simplify usage of `this.tags` without needing to use default value `|| []`, like for example in line 23.

Reply:
This is done so we don't get `tags: []`  on the logs. If tags is undefined only the item is returned.

`info:identifi {"tags":[],"message":"this is a message"}`  // if empty array is set
`info:identifi {"message":"this is a message"}` +2ms // current code behavior

Reply:
👌 👌 👌